src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -32,6 +33,7 @@
 
     // NOTE: keeping the driving source of variants separate from other, supplementary FeatureInputs in our FeatureManager in GATKTool
```

Comment:
This comment is now slightly out of date, since we now do add the driving source of variants to our `FeatureManager` (just with special handling).